pkg/apply/taskrunner/task.go

```diff
@@ -208,34 +198,6 @@ type resourceWaitData struct {
 	generation int64
 }
```

Comment:
I think the `resourceWaitData` struct is no longer in use and can be deleted.

Reply:
done.